test/api-test.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const expect = require('chai').expect;
+const superagent = require('superagent');
+const storage = require('../lib/storage.js');
+
+const apiURL = `http://localhost:${process.env.PORT || 3000}`;
+require('../server.js');
+
+const Athlete = require('./../model/athlete');
+
+describe('Athlete API Routes', function() {
+  it('should create an instanceof Athlete', function(){
```

Comment:
You have an it block nested inside another it block, which isn't the right style. It blocks should be completely separated from each other as they're considered separate tests. Because of this, Mocha considered this as two tests, with one passing, and one failing.